src/Utils.js

```diff
@@ -76,49 +76,61 @@ export const Pollutants = {
         id: "pm25",
         name: "PM25",
         unit: "\xB5g/m\u00B3",
-        safe: "??",
+        baseline: 35,
```

Comment:
Do we have a source for all of these values? might just be good to have handy moving forward

Reply:
Pollutants thresholds: https://docs.google.com/document/d/1DAdqBDDwUHGc4v8tjZyKpxbMy2CmlfGCQtZPeJ7fZ4M/edit